core/src/main/java/org/apache/accumulo/core/fate/Fate.java

```diff
@@ -235,6 +235,9 @@ public Fate(T environment, TStore<T> store, Function<Repo<T>,String> toLogStrFun
    * Launches the specified number of worker threads.
    */
   public void startTransactionRunners(AccumuloConfiguration conf) {
+    if (executor != null) {
+      throw new IllegalStateException("Fate.startTransactionRunners called twice.");
+    }
```

Comment:
Could rely on memozing this, so it can't be created twice, and you can also get the effect of lazily creating it when first used, so you don't need to call a start method to get it running.